wgpu-types/src/lib.rs

```diff
@@ -3440,6 +3458,7 @@ impl TextureFormat {
             Self::Rgb10a2Uint =>          (        msaa, attachment),
             Self::Rgb10a2Unorm =>         (msaa_resolve, attachment),
             Self::Rg11b10Ufloat =>        (        msaa,   rg11b10f),
+            Self::R64Uint =>              (        noaa, attachment),
```

Comment:
This should probably be `TextureUsages::STORAGE_BINDING | TextureUsages::SHADER_ATOMIC`.